### PR TITLE
chore: Add some badges to README

### LINKED ISSUE #24: Add Badges to README

### PR DESCRIPTION
**Pull request by merc1031**
Closes #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,10 @@
 # kintro
+
+[![GitHub license](https://img.shields.io/github/license/Naereen/StrapDown.js.svg)](https://github.com/neckbeard-io/kintro/blob/master/LICENSE)
+[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/neckbeard-io/kintro/main.svg)](https://results.pre-commit.ci/latest/github/neckbeard-io/kintro/main)
+[![CI](https://github.com/neckbeard-io/kintro/actions/workflows/check.yml/badge.svg)](https://github.com/neckbeard-io/kintro/actions/workflows/check.yml)
+[![GitHub latest commit](https://badgen.net/github/last-commit/Naereen/Strapdown.js)](https://GitHub.com/neckbeard-io/kintro/commit/)
```
Comment (brutog) [inline README.md]: This one looks wrong? Unless I'm missing how it's coming up with that date. github.com should probably be case smashed too due to OCD.

Comment (merc1031) [reply README.md]: Changed it up

**Review by brutog — APPROVED** [review]
last commit looks better now. shields.io looks better than that other place.

**Comment by merc1031** [comment]
Yah. shields.io is cool. it just doesnt have all the badges

**Pull request merged**